docs/reference.rst

```diff
@@ -697,6 +697,51 @@ No built-in slot update will run and no hook will be executed.
   ``<key>``.
   ``-`` is converted to ``_`` for use as an environment variable name.
 
+.. _log.logger-sections:
+
+**[log.<logger>] sections**
+
+With a logger section, a *RAUC event logger* can be configured.
```

Comment:
We should point to the section in `advanced.rst` here for the general behavior.

Reply:
Indeed, this did not exist yet when I started here, will fix.